services/ui-src/src/utils/other/export.tsx

```diff
@@ -208,6 +208,9 @@ const sx = {
   },
   entityBox: {
     marginBottom: "1rem",
+    "td, p": {
+      marginBottom: "0.25rem",
+    },
```

Comment:
Note: Design requested that these be wrapped in a list item, but we can wait until we get pdf accessibility feedback on that to see if it will be necessary.